controllers/templates/precache-templates.go

```diff
@@ -136,6 +136,12 @@ spec:
               env:
               - name: config_volume_path
                 value: /tmp/precache/config
+              - name: SPACE_REQUIRED
```

Comment:
@vitus133 Do you think we should add these to the existing config volume instead of more env vars?

Reply:
I think using downwards API with env vars is a clean way of passing info to pods.  It is also easy to template, so in the future, when we determine the precache spec (and the payload size) on the hub, this info will be easy to pass forward.

Reply:
Ok, so we could get rid of the config map and pass everything this way? That would save us an MCV and an MCA?

Reply:
Yes, it's possible. The CM is somewhat more convenient for creating several independent files, where potentially we can have structured text